src/server/tls_util.cc

```diff
@@ -199,23 +199,23 @@ UniqueSSLContext CreateSSLContext(const Config *config, const SSL_METHOD *method
   }
 
   if (SSL_CTX_use_PrivateKey_file(ssl_ctx.get(), config->tls_key_file.c_str(), SSL_FILETYPE_PEM) != 1) {
-    LOG(ERROR) << "Failed to load SSL private key file: " << SSLErrors{};
+    error("Failed to load SSL private key file: {}", SSLErrors{});
     return nullptr;
   }
 
   if (SSL_CTX_check_private_key(ssl_ctx.get()) != 1) {
-    LOG(ERROR) << "Failed to check the loaded private key: " << SSLErrors{};
+    error("Failed to check the loaded private key: {}", SSLErrors{});
     return nullptr;
   }
 
   if (!config->tls_ciphers.empty() && !SSL_CTX_set_cipher_list(ssl_ctx.get(), config->tls_ciphers.c_str())) {
-    LOG(ERROR) << "Failed to set SSL ciphers: " << SSLErrors{};
+    error("Failed to set SSL ciphers: {}", SSLErrors{});
     return nullptr;
   }
 
 #ifdef SSL_OP_NO_TLSv1_3
   if (!config->tls_ciphersuites.empty() && !SSL_CTX_set_ciphersuites(ssl_ctx.get(), config->tls_ciphersuites.c_str())) {
-    LOG(ERROR) << "Failed to set SSL ciphersuites: " << SSLErrors{};
+    error("Failed to set SSL ciphersuites: {}", SSLErrors{});
```

Comment:
```suggestion
    error("Failed to set SSL ciphersuites: {}", fmt::streamed(SSLErrors{}));
```

And maybe in other places..